CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 - Fixed - `Autopopulate.populate` excludes `reserved` jobs in addition to `ignore` and `error` jobs 
 - Fixed - Issue [#1159]((https://github.com/datajoint/datajoint-python/pull/1159)  (cascading delete)  - PR [#1160](https://github.com/datajoint/datajoint-python/pull/1160)
```

Comment:
```suggestion
- Fixed - Issue [#1159](https://github.com/datajoint/datajoint-python/pull/1159) (cascading delete) - PR [#1160](https://github.com/datajoint/datajoint-python/pull/1160)
```

@CBroz1 Just this change. I need to pivot to other tasks today but will work on publishing `0.15.dev0` tomorrow.

Reply:
> publishing `0.15.dev0` tomorrow.

Thanks so much!